workspace/apps/pidp/src/app/features/access/pages/hcim-reenrolment/hcim-reenrolment.page.html

```diff
@@ -22,24 +31,41 @@
     <ui-page-section>
       <ui-page-section-subheader
         icon="assignment"
-        heading="HCIM Web Account Transfer">
+        heading="HCIMWeb Account Transfer">
         <ng-container uiPageSectionSubheaderDesc>
-          You will no longer be able to access HCIM Web through the HealthNetBC
-          portal with your HealthNetBC username and password.
+          HCIMWeb will be changing its login and authentication process with the
+          following key changes:
+          <ul>
+            <li>
+              You will no longer access HCIMWeb through the HealthNetBC Portal,
+              using your HCIMWeb HealthNetBC Username and Password.
+            </li>
+            <li>
+              Instead, you will login directly through
+              <strong>
+                <ui-anchor [href]="hcimWebUrl"> this link</ui-anchor> </strong
+              >, using your organization credential (the same credentials used
+              to access your workstation).
+            </li>
+          </ul>
         </ng-container>
         <p>
-          Instead you will login directly using your organization's credentials
-          (the same credentials used to access your workstation).
+          <strong
+            >Enter your HCIMWeb HealthNetBC Username and Password to transfer
+            your current HCIMWeb HealthNetBC account credential to your
+            organization credential.</strong
+          >
         </p>
-        <p>Complete this step to transition your access.</p>
       </ui-page-section-subheader>
     </ui-page-section>
 
     <ng-container [formGroup]="formState.form">
       <ui-page-section>
-        <ui-page-section-subheader heading="HealthNetBC Credentials">
+        <ui-page-section-subheader heading="HCIMWeb HealthNetBC Username">
```

Comment:
This is a great example of a `ui-page-section-subheader` where it has a small amount of text describing a section that you wouldn't consider section content.